src/pushsource/_impl/backend/staged/staged_ami.py

```diff
@@ -33,6 +33,17 @@ def __push_item(self, leafdir, metadata, entry):
 
         release = AmiRelease(**release_kwargs)
 
+        billing_code_md = attributes.get("billing_code")
+
+        billing_code_attrs = ["name", "code"]
+        billing_code_kwargs = {}
+
+        for key in billing_code_attrs:
+            if key in billing_code_md:
```

Comment:
Am I missing something or is it going to crash if "billing_code" was absent in the metadata file?

It will usually be absent, so then billing_code_md will be None, and 'key in billing_code_md' would crash.

If I'm right: it would have been better to update the test to cover both cases, rather than updating it to cover only the new case of billing_code being present.

Reply:
This is currently on hold, the specification of required functionality was not properly described. Not sure if it should be optional or required.
But you're now it's gonna crash with missing billing code.